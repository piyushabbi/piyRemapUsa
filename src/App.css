.App {
	margin: 0 20px;
}

.App h1 {
  text-align: center;
  text-decoration: underline;
}

.App .states {
	display: flex;
	flex-direction: column;
	align-items: center;
}

.App pre code {
	background-color: #eee;
	border: 1px solid #999;
	display: block;
	padding: 20px;
	overflow-x: scroll;
	margin: 20px 10%;
	text-align: left;
}

.show-json {
	text-align: center;
}

.show-json button {
	cursor: pointer;
	padding: 10px 20px;
	color: #eee;
	background: #222;
}

.show-json button:active {
	background: #555;
}
