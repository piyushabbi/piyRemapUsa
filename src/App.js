import React, { Component } from 'react';
import './App.css';
// import Map from './MapDander';
import MapPage from './components/MapPage';

class App extends Component {
	render() {
		return (
			<div className="App">
				<MapPage />
			</div>
		);
	}
}

export default App;
