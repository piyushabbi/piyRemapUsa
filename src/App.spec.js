import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import App from './App';

describe('<App /> Component', () => {
	it('renders App component', () => {
		const div = document.createElement('div');
		ReactDOM.render(<App />, div);
		expect(ReactDOM.unmountComponentAtNode(div)).toEqual(true);
	});

	it('renders a 1 child Compoent', () => {
		const wrapper = shallow(<App />);
		expect(wrapper.children().length).toEqual(1);
	});

	it('matches snapshot', () => {
		const tree = shallow(<App />);
		expect(toJson(tree)).toMatchSnapshot();
	});
});
