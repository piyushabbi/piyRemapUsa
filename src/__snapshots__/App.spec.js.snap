// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Component matches snapshot 1`] = `
<div
  className="App"
>
  <App />
</div>
`;
